Appraisals

```diff
@@ -1,34 +1,6 @@
 # frozen_string_literal: true
 
-appraise "rubocop-0.60" do
-  gem "rubocop", "~> 0.60"
-  gem "rails", "~> 5.2"
-end
-appraise "rubocop-0.61" do
-  gem "rubocop", "~> 0.61"
-  gem "rails"
-end
-appraise "rubocop-0.62" do
-  gem "rubocop", "~> 0.62"
-  gem "rails"
-end
-appraise "rubocop-0.63" do
-  gem "rubocop", "~> 0.63"
-  gem "rails"
-end
-appraise "rubocop-0.64" do
-  gem "rubocop", "~> 0.64"
-  gem "rails"
-end
-appraise "rubocop-0.65" do
-  gem "rubocop", "~> 0.65"
-  gem "rails"
-end
-appraise "rubocop-0.66" do
-  gem "rubocop", "~> 0.66"
-  gem "rails"
-end
-appraise "rubocop-0.67" do
-  gem "rubocop", "~> 0.67"
+appraise "rubocop-0.68" do
+  gem "rubocop", "~> 0.68"
```

Comment:
I think that it is good to add RuboCop Performance.

```ruby
gem "rubocop-performance"
```

Reply:
@koic Do you think we should also add `rubocop-performance` as the gem's runtime dependency?

Reply:
@toshimaru Sure! It is reasonable to add to gemspec.

Reply:
Done!